helper/common/common.go

```diff
@@ -93,3 +93,8 @@ func GetTerminationSignalCh() <-chan os.Signal {
 
 	return signalCh
 }
+
+// GetDNSRegex returns a regex string  to match against a valid dns/dns4/dns6 addr
+func GetDNSRegex() string {
+	return `^/?(dns)(4|6)?/[^-|^/][A-Za-z0-9-]([^-|^/]?)+([\\-\\.]{1}[a-z0-9]+)*\\.[A-Za-z]{2,}(/?)$`
+}
```

Comment:
You might want to define constant value instead of function?

Reply:
For now, we don't need to put this under `helper`. Why don't you define in network/server.go?

Reply:
Resolved here: https://github.com/0xPolygon/polygon-edge/pull/406/commits/2ad796dbf3e4884a9b15f72e9fa9059d88744282